lib/strtoday.c

```diff
@@ -19,20 +19,6 @@
 #include "string/strspn/stpspn.h"
 
 
-/*
- * strtoday() now uses get_date() (borrowed from GNU shellutils)
- * which can handle many date formats, for example:
- *	1970-09-17	# ISO 8601.
- *	70-9-17		# This century assumed by default.
```

Comment:
Funnily enough, `This` century is an interesting implicit assumption that the comment wouldn't survive *that* century.  :D